src/main/resources/application.yml

```diff
@@ -32,6 +32,7 @@ spring:
     virtualhost: /
 
 queueconfig:
+  telephone-capture-queue: caseApi.case.telephoneCapture
```

Comment:
should this be `caseApi.caseProcessor.telephoneCapture` if we're using the new naming standard?

Reply:
Fair point, best to be completely explicit with the service names I suppose